src/bin/travis/testCentaurJes.sh

```diff
@@ -26,19 +26,36 @@ printTravisHeartbeat() {
     TRAVIS_HEARTBEAT_PID=$!
 }
 
+
+cromwellLogTail() {
+ (
+   while [ ! -f log.txt ];
+   do
+     sleep 2
+     printf "(Cr)"
+   done
+   tail -f log.txt &
+   CROMWELL_LOG_TAIL_PID=$!
+ ) &
+ CROMWELL_LOG_WAIT_PID=$!
+}
+
 killTravisHeartbeat() {
     if [ -n "${TRAVIS_HEARTBEAT_PID+set}" ]; then
         kill ${TRAVIS_HEARTBEAT_PID} || true
     fi
 }
 
 exitScript() {
+    echo "CROMWELL LOG"
+    cat log.txt
     killTravisHeartbeat
```

Comment:
Are these changes relevant to this PR?  It looks like useful stuff, I'm just not sure why it's her.